lisa/sut_orchestrator/azure/common.py

```diff
@@ -1696,6 +1698,7 @@ def get_or_create_storage_container(
     """
     Create a Azure Storage container if it does not exist.
     """
+    credential = get_static_access_token("AZURE_STORAGE_ACCESS_TOKEN") or credential
```

Comment:
Why the env name variable is different? Is it defined by the staroge class or us?

Reply:
it is defined by us, the tokens need to be generated based on different scope.

Reply:
1. Is it possible to have one token with all scopes like *?
2. Define schema to accept tokens, instead of env vars. All LISA vars should be defined in runbook.

Let me know, if it's hard to fulfill above.

Reply:
1. I have confirmed with the maintainer of python sdk, he said we have to use different tokens for different scopes.
2. Use different variables to accept the tokens?

Reply:
> 2. Use different variables to accept the tokens?

As I said, define in schema for them, not directly use env vars. After you defined schema, you can set values by LISA vars, and then assign LISA vars by env vars.

Like you defined `access_token` in PlatformSchema. Please define similar tokens for other Azure scopes.